support/ebpf/dotnet_tracer.ebpf.c

```diff
@@ -44,43 +44,48 @@ bpf_map_def SEC("maps") dotnet_procs = {
 // currently not Garbage Collected by the runtime. Though, we have submitted also an enhancement
 // request to fix the nibble map format to something sane, and this might get implemented.
 // see: https://github.com/dotnet/runtime/issues/93550
-static inline __attribute__((__always_inline__))
-ErrorCode dotnet_find_code_start(PerCPURecord *record, DotnetProcInfo *vi, u64 pc, u64 *code_start) {
+static inline __attribute__((__always_inline__)) ErrorCode
+dotnet_find_code_start(PerCPURecord *record, DotnetProcInfo *vi, u64 pc, u64 *code_start)
+{
   // This is an ebpf optimized implementation of EEJitManager::FindMethodCode()
   // https://github.com/dotnet/runtime/blob/v7.0.15/src/coreclr/vm/codeman.cpp#L4115
   // The support code setups the page mapping so that:
   //   text_section_base = pHp->mapBase (base address of the JIT area)
   //   text_section_id   = pHp->pHdrMap (pointer to the nibble map)
   const UnwindState *state = &record->state;
   DotnetUnwindScratchSpace *scratch = &record->dotnetUnwindScratch;
-  const int map_elements = sizeof(scratch->map)/sizeof(scratch->map[0])/2;
+  const int map_elements = sizeof(scratch->map) / sizeof(scratch->map[0]) / 2;
   u64 pc_base = state->text_section_bias;
   u64 pc_delta = pc - pc_base;
   u64 map_start = state->text_section_id;
 
-  DEBUG_PRINT("dotnet:  --> find code start for %lx: pc_base %lx, map_start %lx",
-    (unsigned long) pc_delta, (unsigned long) pc_base, (unsigned long) map_start);
-  pc_delta &= ~(DOTNET_CODE_ALIGN-1);
+  DEBUG_PRINT(
+      "dotnet:  --> find code start for %lx: pc_base %lx, map_start %lx",
+      (unsigned long)pc_delta,
+      (unsigned long)pc_base,
+      (unsigned long)map_start);
```

Comment:
4 spaces here doesn't match with overall 2 spaces indentation.
Same at many other places (I won't comment on more).

Reply:
I didn't request a manual change. It's clear how clang-format works. There should be an option to get this right.

Reply:
So it is: `ContinuationIndentWidth: 2`